masa/api/validator_api.py

```diff
@@ -117,45 +117,54 @@ def __init__(self, validator, config=None):
         self.start_server()
 
     async def get_twitter_profile(self, username: str):
-        return await TwitterProfileForwarder(self.validator).forward_query(
+        all_responses = await TwitterProfileForwarder(self.validator).forward_query(
             query=username
         )
+        return all_responses[0]
 
     async def get_twitter_followers(self, username: str):
-        return await TwitterFollowersForwarder(self.validator).forward_query(
+        all_responses = await TwitterFollowersForwarder(self.validator).forward_query(
             query=username
         )
+        return all_responses[0]
 
     async def get_recent_tweets(self, tweet_query: RecentTweetsQuery):
-        return await TwitterTweetsForwarder(self.validator).forward_query(
+        all_responses = await TwitterTweetsForwarder(self.validator).forward_query(
             tweet_query=tweet_query
         )
+        return all_responses[0]
 
     async def scrape_web(self, web_scraper_query: WebScraperQuery):
-        return await WebScraperForwarder(self.validator).forward_query(
+        all_responses = await WebScraperForwarder(self.validator).forward_query(
             web_scraper_query=web_scraper_query
         )
+        return all_responses[0]
 
     async def get_discord_profile(self, user_id: str):
-        return await DiscordProfileForwarder(self.validator).forward_query(
+        all_responses = await DiscordProfileForwarder(self.validator).forward_query(
             query=user_id
         )
+        return all_responses[0]
```

Comment:
Don't see the point of turning it into array and then indexing first element if it was already working before. Is it due to forwarder change or something else @obasilakis?

Reply:
yeap